expo/eas-cli
# Adds workflow:validate command

# Why

This adds a command to validate YAML in workflow configuration files.

# Screenshots

## Valid YAML output
<img width="420" alt="Screenshot 2024-10-24 at 12 04 22 AM" src="https://github.com/user-attachments/assets/0c08143b-6db2-463e-8653-a11803eda405">

## Invalid YAML output
<img width="534" alt="Screenshot 2024-10-24 at 12 04 28 AM" src="https://github.com/user-attachments/assets/3de1a769-faff-49f0-83f1-3de40ea8010a">



# Test Plan

Create a .yml file and test it with `eas workflow:validate path/to/file.yml`. You should see output like the screenshots above.
- Make sure non existent files result in an error
- Make sure non .yml/.yaml files result in an error
- Make sure bad YAML results in an error
- Make sure good YAML results in no error
